guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -406,6 +406,16 @@ You can now create an image from this virtual machine.
 
 You can use the xref:Provisioning_Virtual_Machines_in_VMware_vSphere-Adding_VMware_vSphere_Images_on_the_Satellite_Server[] section to add the image to {Project}.
 
+.Configuring {SmartProxy} to forward user data template
+
+{Project} must recognize the client IP address in order to serve correct template payload and when clients connects directly to {ProjectServer}, remote IP of the HTTP request can be used. However, when {Project} is deployed with {SmartProxy} templates feature, the callback URL is set to {SmartProxy} endpoint which forwards to the {ProjectServer} with HTTP header information X-Forwarded-For.
+
+For security reasons, {Project} does recognize this HTTP header only from localhost. For each individual {SmartProxy}, a regular expression in *Administer* > *Settings* > *Provisioning* > *Remote address* must be updated to recognize IP address. From CLI this can be done using the following command:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer settings set --name remote_addr --value '(192.168.122.(1|2|3)'
```

Comment:
It's a regex, so dots should be escaped. You also miss a closing brace. We should also recommend beginning and end matchers. The example would also match `192.168.122.12`

Also please use IPs from example ranges: https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600.

```suggestion
# hammer settings set --name remote_addr --value '^192\.0\.2\.(1|2|3)$'
```